packages/orchestrator-ui-components/src/components/WfoFilterTabs/styles.ts

```diff
@@ -0,0 +1,13 @@
+import { css } from '@emotion/react';
+
+import { WfoTheme } from '@/hooks';
+
+export const getStyles = ({ theme }: WfoTheme) => {
+    const tabStyle = css({
+        '.euiTab__append': {
+            marginLeft: `-${theme.size.xs}`,
```

Comment:
I think we should avoid negative margins and paddings and such as much as possible. `discussion`